.github/workflows/e2e.yaml

```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       matrix:
        kube-version:
+       - "1.19" # long term support for OpenShift 4.6 based on k8s 1.19
```

Comment:
This looks opinionated to the RH use case. 

WDYT @jpkrohling?

Reply:
I am removing this.